main_page.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Should this come before the doctype?

Reply:
Think so - this page has been trainer reviewed :)